samples/CustomRouteSample.Web/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace CustomRouteSample.Web.Controllers
```

Comment:
What I don't like is that they overlap. Both of theses are related to routing. It's simpler if they are separate.